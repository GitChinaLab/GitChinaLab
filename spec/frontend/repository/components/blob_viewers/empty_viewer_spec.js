import { shallowMount } from '@vue/test-utils';
import EmptyViewer from '~/repository/components/blob_viewers/empty_viewer.vue';

describe('Empty Viewer', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallowMount(EmptyViewer);
  });

  it('matches the snapshot', () => {
    expect(wrapper.element).toMatchSnapshot();
  });
});
