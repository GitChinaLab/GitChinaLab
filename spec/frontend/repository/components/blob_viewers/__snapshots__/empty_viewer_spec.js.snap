// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Empty Viewer matches the snapshot 1`] = `
<div
  class="nothing-here-block"
>
  Empty file
</div>
`;
